packages/filebrowser/src/listing.ts

```diff
@@ -806,6 +809,100 @@ export class DirListing extends Widget {
     }
   }
 
+  // Update the modified column's size
+  private _updateModifiedSize(node: HTMLElement) {
+    const modified = DOMUtils.findElement(node, ITEM_MODIFIED_CLASS);
+    this._modifiedWidth = modified?.getBoundingClientRect().width ?? 83;
+    this._modifiedStyle =
+      this._modifiedWidth < 90
+        ? 'narrow'
+        : this._modifiedWidth > 110
+        ? 'long'
+        : 'short';
+  }
+
+  // Update only the modified dates.
+  protected updateModified(items: Contents.IModel[], nodes: HTMLElement[]) {
+    items.forEach((item, i) => {
+      const node = nodes[i];
+      if (item.last_modified) {
+        const modified = DOMUtils.findElement(node, ITEM_MODIFIED_CLASS);
+        if (this.renderer.updateItemModified !== undefined) {
+          this.renderer.updateItemModified(
+            modified,
+            item.last_modified,
+            this._modifiedStyle
+          );
+        } else {
+          // Render the date in one of multiple formats, depending on the container's size
+          modified.textContent = Time.formatHuman(
+            item.last_modified,
+            this._modifiedStyle
+          );
+          modified.title = Time.format(new Date(item.last_modified));
```

Comment:
I think this can be simplified; the fallback to the `defaultRenderer` is the pattern we use frequently when introducing new optional methods:

```suggestion
          DirListing.defaultRenderer.updateItemModified(
            modified,
            item.last_modified,
            this._modifiedStyle
          );
```